docs/source/index.rst

```diff
@@ -42,6 +42,15 @@ PyTorch Lightning
    common/lightning_module
    common/trainer
 
+.. toctree::
+   :maxdepth: 1
+   :name: Accelerators
+   :caption: Accelerators
+
+   accelerators/gpu
+   accelerators/tpu
```

Comment:
The index could be further expanded to

* GPU
	* Single GPU Training
	* Multi GPU Training
	* Multi Node Training
* TPU
	* Single TPU Training
	* TPU Pod Training
....

Reply:
I like this.

Reply:
@awaelchli Here's the notion [page](https://www.notion.so/gridai/Lightning-docs-revamp-1853e181cf334c10a43ae963c5f1d33a). I will update #7666 and clean the notion page a bit more. 

The rationale for creating a distinct Accelerators section is to increase visibility and group them together for users to investigate. It was more difficult to look for different accelerators in the Common Cases section as they were listed in a different order and felt disorganized.

Reply:
The main priority right now would be sorting out the Getting Started section. It's a bit messy right now

```python
### Getting Started

1. Installation
2. Quick Start
    1. Intro 
    2. Prepare Data (Data Module) 
    3. Research Code (Lightning Module) 
    4. Abstract Boilerplate (Trainer) 
    5. Maximum Flexibility (Callbacks)
    6. Starter Templates
3.  From PyTorch to Lightning
```

Reply:
Created an issue https://github.com/PyTorchLightning/pytorch-lightning/issues/10765